APMrover2/control_modes.cpp

```diff
@@ -110,53 +107,82 @@ void Rover::reset_control_switch()
     read_control_switch();
 }
 
-// read at 10 hz
-// set this to your trainer switch
-void Rover::read_trim_switch()
+// ready auxiliary switch's position
+aux_switch_pos Rover::read_aux_switch_pos()
+{
+    uint16_t radio_in = channel_aux->get_radio_in();
+    if (radio_in < AUX_SWITCH_PWM_TRIGGER_LOW) return AUX_SWITCH_LOW;
```

Comment:
Thanks for the review.  The other vehicles do it this way as well so although it's technically against our style policy, i'm going to leave it consistent with the other vehicles.